src/web/routes/application/email-address/validate.test.js

```diff
@@ -0,0 +1,50 @@
+const test = require('tape')
+
+const { EMAIL_ADDRESS_REGEX } = require('./validate')
+
+test('valid email', (t) => {
+  const emailAddresses = [
+    'email@domain.com',
+    'firstname.lastname@domain.com',
+    'email@subdomain.domain.com',
+    'firstname+lastname@domain.com',
+    '1234567890@domain.com',
+    'email@domain-one.com',
+    '_______@domain.com',
+    'email@domain.name',
+    'email@domain.co.jp',
+    'firstname-lastname@domain.com']
```

Comment:
Can we add some non-ascii characters, e.g. cyrillic?

Reply:
Done.